modules/disk-uploader/pkg/parse/clioptions.go

```diff
@@ -18,7 +18,7 @@ type CLIOptions struct {
 	ExportSourceName      string `arg:"--export-source-name" help:"Name of the export source"`
 	VolumeName            string `arg:"--volumename" help:"Name of the volume (if source kind is 'pvc', then volume name is equal to source name)"`
 	ImageDestination      string `arg:"--imagedestination" help:"Destination of the image in container registry"`
-	PushTimeout           int    `arg:"--pushtimeout" help:"Push timeout of container disk to registry"`
+	PushTimeout           string `arg:"--pushtimeout" help:"Push timeout of container disk to registry"`
```

Comment:
Why did you change the timeout to string?

Reply:
Tekton expects only string parameters and there is no integer types.